app/profile/test/test.html

```diff
@@ -31,8 +31,8 @@
 
         <div class="ibox">
             <div class="ibox-title">
-                <div class="ibox-tools">
-                    <a class="btn btn-white" ng-click="vm.taskFormModal()">Додати ще</a>
+                <div class="ibox-tools" ng-if="vm.isCreateNewTaskPermission()">
```

Comment:
Hmm... I see, you try to hide the content from non-admin users, and show it for admin only. We already have "user role directives" for this. Directives has several examples of how to use it. You can find it at ./app/utils/directives/user.role.directive.js